### DISCUSSION TITLE
The GitHub Action status has been stuck at Queued

### DISCUSSION THREAD

**Discussion by Yinzhuo19970516** [community/community#50688] [Actions]
### Select Topic Area

Bug

### Body

The GitHub Action status has been stuck at Queued for about 5 minutes and then changed to Startup failure. What could be the reason for this?
https://github.com/Yinzhuo19970516/vue-template/actions

**Comment by Akash1134** [comment]
Hi @Yinzhuo19970516 , there are no current [service outages](https://githubstatus.com/) reported for GitHub Actions. I'd recommend re-running the action to see if that helps.

**Reply by Yinzhuo19970516** [reply]
oh， I have tried many times already.
